src/holochain/app_conn_pool.rs

```diff
@@ -328,9 +328,35 @@ impl AppCall for AppConnPool {
     fn handle_zome_call(
         &self,
         installed_app_id: InstalledAppId,
-        execute: fn(AppWebsocket) -> BoxFuture<'static, HcHttpGatewayResult<ExternIO>>,
+        cell_id: CellId,
+        zome_name: String,
+        fn_name: String,
+        payload: ExternIO,
     ) -> BoxFuture<'static, HcHttpGatewayResult<ExternIO>> {
         let this = self.clone();
-        Box::pin(async move { this.call(installed_app_id, execute).await })
+        Box::pin(async move {
+            this.call(installed_app_id, |app_ws| {
+                let cell_id = cell_id.clone();
+                let zome_name = zome_name.clone();
+                let fn_name = fn_name.clone();
+                let payload = payload.clone();
+                Box::pin(async move {
+                    let result = app_ws
+                        .call_zome(
+                            ZomeCallTarget::CellId(cell_id),
+                            zome_name.into(),
+                            fn_name.into(),
+                            payload,
+                        )
+                        .await;
+                    if let Err(err) = &result {
+                        tracing::debug!(?err);
```

Comment:
Shouldn't this be an error log?

Reply:
There could potentially be many of these and the administrator can't do much about it if the app doesn't have the data that's being requested or is coded wrong. At most, we would monitor the gateway for an error response count and come looking at what's going on if that spikes